src/main/java/fr/techad/edc/popover/internal/swing/components/Popover.java

```diff
@@ -231,7 +238,12 @@ private JComponent getHeader() {
         header.setBackground(contentPanel.getBackground());
         header.setLayout(new FlowLayout(FlowLayout.RIGHT, 2, 2));
         ImageIcon imageIcon = ImageIconCreator.createImageIcon(iconPath);
-        IconButton closeButton = new IconButton("Close", imageIcon);
```

Comment:
you can implement the IconButton like this:
IconButton closeButton = new IconButton(this.displayTooltip?"Close":null, imageIcon);